gradle.properties

```diff
@@ -2,6 +2,7 @@ kotlin.code.style=official
 kotlin.incremental.js=true
 kotlin.incremental.multiplatform=true
 kotlin.mpp.stability.nowarn=true
+kotlin.native.binary.sourceInfoType=libbacktrace
```

Comment:
Were you able to get better stack traces with this enabled? I remember you said it didn't help, so I'd rather leave it off if it's not working as expected

Reply:
Good point, will remove. I eventually discovered that full stack traces _are_ available in the build artifacts (e.g., **build/reports/tests/linuxX64Test/index.html**) just not in the console output.